roles/deluge/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Create Deluge Directories
+  file:
+    path: "{{ item }}"
+    state: directory
+    # mode: 0755
+  with_items:
+    - "{{ deluge_config_directory }}"
+    - "{{ deluge_download_directory }}"
+
+- name: Deluge Docker Container
+  docker_container:
+    name: deluge
+    image: linuxserver/deluge
+    pull: true
+    volumes:
+      - "{{ deluge_config_directory }}:/config:rw"
+      - "{{ deluge_download_directory }}:root/Downloads:rw"
```

Comment:
Per the the linuxserver docs default downloads path within the container is `/downloads`

Reply:
Code snippit from [linuxserver/docker-deluge/readme-vars.yml](https://github.com/linuxserver/docker-deluge/blob/master/readme-vars.yml)
```yml
app_setup_block: |
  The admin interface is available at `http://SERVER-IP:8112` with a default user/password of admin/deluge.
  To change the password (recommended) log in to the web interface and go to Preferences->Interface->Password.
  Change the downloads location in the webui in Preferences->Downloads and use /downloads for completed downloads.
```
Deluge's default config points the downloads folder to /root/Downloads and I haven't found another way to set it via task yet. I changed it so the end user doesn't have to do that final step to hopefully make using deluge a smoother experience. If you want to adhere to the defaults we can change it back, but users will need to verify their downloads folder after first start up.